### PR TITLE
dircolors: implement --print-ls-colors

### PR DESCRIPTION
**Pull request by cakebaker**

### CODE REVIEW

File: src/uu/dircolors/src/dircolors.rs [hunk 1]
```diff
@@ -262,11 +282,12 @@ where
     T: IntoIterator,
     T::Item: Borrow<str>,
 {
-    // 1440 > $(dircolors | wc -m)
-    let mut result = String::with_capacity(1440);
+    // 1790 > $(dircolors | wc -m)
+    let mut result = String::with_capacity(1790);
```
Comment (tertsdiepraam) [inline src/uu/dircolors/src/dircolors.rs:286]: Where does this change come from? Does is have anything to do with the change you made or did you just find a different value?

**Review by tertsdiepraam — COMMENTED** [review]
Works like a charm! I just have a small question about the string allocation.

File: src/uu/dircolors/src/dircolors.rs (hunk 1 above)
Comment (cakebaker) [reply src/uu/dircolors/src/dircolors.rs:286]: This is a change related to #3559 I didn't notice back then. The value is rounded up from what `dircolors | wc -m` returns (1784).

Comment (tertsdiepraam) [reply src/uu/dircolors/src/dircolors.rs:286]: Alright thanks!

**Pull request merged**